### ISSUE TITLE
Save open manuscript in current session

### ISSUE THREAD

**Issue by uliska** [wbsoft/frescobaldi#715]
Pulled out from #711.

If the current session is configured to automatically save opened documents the opened manuscript should be stored/restored too.

**Comment by uliska** [comment]
OK, it has to be in `sessions.manager.saveCurrentSession()`.

The question is: should we reference something like `mainwindow().manuscriptviewer().documents` there or should we add `mainwindow().manuscripts` and `mainwindow().currentManuscript()` in analogy to the input documents?

**Comment by uliska** [comment]
Hm, I tried to implement it but completely fail so far.

I'm trying to do something like

```python
            manuscripts = panelmanager.manager(self.mainwindow()).manuscript
            print manuscripts.widget()._currentManuscript
```

in `session.manager.saveSession()` (with _currentManuscript being set when the manuscript document is loaded.
But when saving the session it always returns `None`.
While the above returns a `ManuscriptViewPanel` as `manuscripts` it seems to be a different object than that where the document is actually loaded into:

```
<viewers.manuscript.widget.Widget object at 0x7f8ec3123df8>
<viewers.manuscript.widget.Widget object at 0x7f8ec5510770>
```
is the output of printing `self` in the document loading routine and when retrieving `_currentManuscript` later

**Comment by uliska** [comment]
Will `panelmanager.manager(self.mainwindow()).manuscript` return a new object compared to the previous one?

**Comment by PeterBjuhr** [comment]
The panel was loaded twice, see 053d166a05ff761e9fcf72c5c246884afac7e521. Please try it again.

**Comment by uliska** [comment]
That seems to explain the strange behaviour.
Unfortunately I'll only be able to check it out tomorrow. The power
supply of my laptop is broken, and I can only boot the desktop into
Linux when I'm alone at home ;-)

**Comment by uliska** [comment]
Thanks, it's working now, and I'm working on it.

**Comment by PeterBjuhr** [comment]
Great!

**Comment by uliska** [comment]
Implemented the session handling in 1db340c (and previous)

**Comment by PeterBjuhr** [comment]
One observation: If a session is closed and reopened the zoom factor is remembered but not the current page. This is consistent with the behavior of the music view which is good. But perhaps it would be nice if the page also is remembered??
